ui/app/src/Utils/Views.elm

```diff
@@ -1,19 +1,28 @@
-module Utils.Views exposing (..)
+module Utils.Views exposing (buttonLink, checkbox, error, formField, formInput, iconButtonMsg, labelButton, linkifyText, loading, tab, textField, validatedField)
 
 import Html exposing (..)
 import Html.Attributes exposing (..)
-import Html.Events exposing (onCheck, onInput, onClick, onBlur)
-import Utils.FormValidation exposing (ValidationState(..), ValidatedField)
+import Html.Events exposing (onBlur, onCheck, onClick, onInput)
+import Utils.FormValidation exposing (ValidatedField, ValidationState(..))
 import Utils.String
 
 
 tab : tab -> tab -> (tab -> msg) -> List (Html msg) -> Html msg
-tab tab currentTab msg content =
+tab tab_ currentTab msg content =
```

Comment:
Just for my understanding, what is the convention in `<variable-name>_`?

Reply:
Variable shadowing is disallowed in 0.19. I've seen folks doing this instead of primes `tab'`.

Reply:
I had that question too. If that's how elm is handling it (I think `shadow'` was carried over from haskell?), that's fine by me.

Reply:
In some cases it is just hard to come with the names. Probably a function has to become a verb, like `viewTab`. We can change this as we work on it in the future.